values.yaml

```diff
@@ -42,30 +46,22 @@ placeholder:
   maxSurge: 1
   maxUnavailable: 0
   minReadySeconds: 5
-  storeURL: "https://s3.amazonaws.com/pelias-data.nextzen.org/portland-metro/placeholder/store.sqlite3.gz"
+  storeURL: "https://data.geocode.earth/placeholder/2020-05-08/store.sqlite3.gz"
   cpus: 1 # how many CPUs to allow using via the npm `cluster2` module
   retries: 1 # number of time the API will retry requests to placeholder
   timeout: 5000 # time in ms the API will wait for placeholder responses
   downloadCommand: >
       mkdir -p /data/placeholder/ &&
       wget -O- "$DOWNLOAD_URL" | gunzip > /data/placeholder/store.sqlite3
-  limits:
-    # cannot use a hyphen to match the k8s param due to https://github.com/helm/helm/issues/2192
-    ephemeral_storage: 5Gi
   requests:
     memory: 512Mi
     cpu: 0.1
-    # cannot use a hyphen to match the k8s param due to https://github.com/helm/helm/issues/2192
```

Comment:
keep original. overriding values will be in another file that should not be committed to this repo